storage/client_merge_test.go

```diff
@@ -227,6 +227,25 @@ func TestStoreRangeMergeNonConsecutive(t *testing.T) {
 		log.Errorf("split ranges keys are equal %q!=%q", rangeA.Desc().StartKey, rangeC.Desc().StartKey)
 	}
 
+	// Read all of the keys touched by the split transactions to resolve
+	// intents.  If intents are left unresolved then the asynchronous
+	// resolution may happen after the call to RemoveRange below,
+	// reviving the range and breaking the test.
+	// Unfortunately we must list those keys explicitly, since there are
+	// encoding problems with using Scan with system keys (#437).
+	// TODO(bdarnell): when we can use Scan with system keys, do so.
```

Comment:
You should be able to scan system keys...

Also, should the TODO here be that we eventually want to split and then rebalance the range elsewhere so the test is more legit?